cmd/register/register_util.go

```diff
@@ -468,32 +454,34 @@ func getJSONSpec(message proto.Message) string {
 	return jsonSpec
 }
 
-func getFlyteTestManifest(org, repository string) ([]FlyteSnack, string, error) {
+func filterExampleFromRelease(releases github.RepositoryRelease) []github.ReleaseAsset {
+	var assets []github.ReleaseAsset
+	for _, v := range releases.Assets {
+		if strings.HasSuffix(*v.Name, ".tgz") {
```

Comment:
Introduce a function instead which checks whether a file is an archive and put that as a common until function

Reply:
No, I think currently it only detects files that are published by us i.e `.tgz` extension, We will ignore the zip extension.